### ISSUE TITLE
Cannot read properties Error

### ISSUE THREAD

**Issue by Snipy44** [renzuzu/renzu_hud#112]
Hello do you say how to fix this ? thank's :)

CrBrowserMain/ Uncaught TypeError: Cannot read properties of null (reading 'style') (@renzu_hud/ui/js/demo.js:788)

**Comment by renzuzu** [comment]
this is some left over elements , that i removed from HUD. the vehicle mode. eco, etc.
this wont affect the UI.

**Comment by Snipy44** [comment]
Yes, but its spam in the F8 menu and I can't keep good visibility
